content/docs/concepts/authorization/v2.x/configuration/_index.md

```diff
@@ -57,20 +57,24 @@ Afterwards, the storage administrator can configure Authorization with the follo
 
 The storage types supported are `powerflex`, `powermax`, and `powerscale`. During the creation of a storage system and role, the storage type must be one of the supported types.
 
-A `storage` entity in Container Storage Modules Authorization consists of the storage type (`powerflex`, `powermax` or `powerscale`), the system ID, the API endpoint, and the vault credentials path. Edit these parameters in the manifest:
+A `storage` entity in Container Storage Modules Authorization consists of the storage type (`powerflex`, `powermax` or `powerscale`), the system ID, the API endpoint, and the storage system credentials.
+
+>__Note__:
+> - You must specify exactly one option for the storageSystemCredentials field, e.g. secretProviderClass.
+
+Edit these parameters in the manifest:
 
    | Parameter                 | Description                                                                          | Required | Default |
    | ------------------------- | ------------------------------------------------------------------------------------ | -------- | ------- |
-   | type                      | The type of the storage array.                                                        | Yes      | -       |
+   | type                      | The type of the storage array.                                                       | Yes      | -       |
    | endpoint                  | HTTPS REST API endpoint of the backend storage array.                                | Yes      | -       |
    | systemID                  | System ID of the backend storage array.                                              | Yes      | -       |
-   | vault.identifier          | The identifier of the Vault to be used that was configured in the Authorization CR.  | Yes      | -       |
-   | vault.kvEngine            | The mount path of the KV secrets engine.                                             | Yes      | secret  |
-   | vault.path                | The location within the store that the credentials for the array are stored.         | Yes      | -       |
+   | storageSystemCredentials.secretProviderClass          | The secretProviderClass containing the storage system credentials.  | Yes     | -       |
+   | storageSystemCredentials.secret            | The Kubernetes Secret containing the storage system credentials     | No      | -       |
    | skipCertificateValidation | A boolean that enables/disables certificate validation of the backend storage array. | No       | true    |
    | pollInterval              | PollInterval is the polling frequency to test the storage connectivity.              | No       | 30s     |
 
-For example, to create PowerFlex storage:
+For example, to create PowerFlex storage using a secret provider class:
```

Comment:
Can we use tabs (tab for Secret, tab for SecretProviderClass) here? See https://dell.github.io/csm-docs/docs/tooling/cert-csi/#download-release-linux "Pull The Container Image".

Reply:
Fixed